src/image.rs

```diff
@@ -72,3 +77,23 @@ pub async fn is_present_locally_podman(images: PodmanImages, search_for_image: S
     }
     false
 }
+
+#[cfg(test)]
```

Comment:
Should this live in the `tests` directory?

Reply:
I think its idiomatic to have unit tests to live alongside the code its testing in Rust

Reply:
Ah right, those are E2E tests. Good call.